traits/src/lib.rs

```diff
@@ -336,9 +336,9 @@ pub trait OnRedundantCall<AccountId> {
 }
 
 pub trait OnReceived<AccountId, CurrencyId, Balance> {
-	fn on_received(account: AccountId, currency: CurrencyId, amount: Balance);
+	fn on_received(account: &AccountId, currency: CurrencyId, amount: Balance);
 }
 
 impl<AccountId, CurrencyId, Balance> OnReceived<AccountId, CurrencyId, Balance> for () {
-	fn on_received(_: AccountId, _: CurrencyId, _: Balance) {}
+	fn on_received(_: &AccountId, _: CurrencyId, _: Balance) {}
```

Comment:
No need to implement for `()`

https://github.com/open-web3-stack/open-runtime-module-library/blob/7b881ef34194ddc93d2a07f4cb10942bb99f935e/traits/src/lib.rs#L333